PittAPI/course.py

```diff
@@ -111,7 +111,7 @@ def _extract_course_data(header, course):
 
 def get_class_description(term, class_number):
     """Return a string that is the description for class in a term"""
-    page = requests.get(URL + 'detail.asp?TERM={}&CLASSNUM={}'.format(_validate_term(term), class_number))
+    page = requests.get(URL + 'detail.asp', params={'TERM': _validate_term(term), 'CLASSNUM': class_number})
```

Comment:
I'd prefer this to not be in-lined, just make a dict called `payload` right above it. See: https://github.com/Pitt-CSC/PittAPI/blob/master/PittAPI/people.py#L31